.travis.yml

```diff
@@ -1,20 +1,23 @@
 language: php
 
-dist: trusty
+dist: xenial
+
+services:
+  - mysql
 
 env:
   global:
-    - COMPOSER_ROOT_VERSION=1.x-dev
+    - COMPOSER_ROOT_VERSION=1.2.x-dev
```

Comment:
Should probably not revert the travis file changes

Reply:
This branch is meant to target the 1.2.x release line. I'll switch it back to 1.x-dev on the merge up.